modules/3d-tiles/src/tiles-3d-loader.js

```diff
@@ -72,7 +72,8 @@ const Tiles3DLoader = {
       decodeQuantizedPositions: false,
       isTileset: 'auto',
       tile: null,
-      tileset: null
+      tileset: null,
+      assetGltfUpAxis: null
```

Comment:
- here it is in `3d-tiles` options, not `cesium-ion`.

- Normally it should be extracted from tileset.json / assets?

Reply:
Added one more "dummy" tileset with Z default axis.
Called it BatchedColorsZUp.
It's only for parser testing.